tests/testthat/test-slice.R

```diff
@@ -191,7 +191,7 @@ test_that("vec_as_location() checks type", {
   expect_error(vec_as_location("foo", "bar"), class = "vctrs_error_incompatible_type")
   expect_error(vec_as_location("foo", 1L, names = 1L), "must be a character vector")
   expect_error(vec_as_location(Sys.Date(), 3L), class = "vctrs_error_subscript_type")
-  expect_error(vec_as_location(matrix(TRUE, nrow = 1), 3L), "must have one dimension")
+  expect_error(vec_as_location(matrix(TRUE, nrow = 1), 3L), "must be a one-dimensional")
```

Comment:
Can you add this to the error output file please? And wrap this test in `verify_errors()`.